test/hotspot/jtreg/compiler/ciReplay/CiReplayBase.java

```diff
@@ -155,8 +155,8 @@ public void cleanup() {
         removeFromCurrentDirectoryStartingWith("core");
         removeFromCurrentDirectoryStartingWith("replay");
         removeFromCurrentDirectoryStartingWith(HS_ERR_NAME);
-        remove(TEST_CORE_FILE_NAME);
-        remove(REPLAY_FILE_NAME);
+        remove(TEST_CORE_FILE_NAME);x
+        //remove(REPLAY_FILE_NAME);
```

Comment:
Change by accident?

Reply:
Yes, I changed it by accident. I've reverted the change.